public/download.php

```diff
@@ -9,64 +14,71 @@
 <div id="mainpage">
     <div id='fullcontainer'>
 
-        <h2>Downloads</h2>
-
-        <p class="embedded mb-5">
-            <b>Upgrade:</b> If you were brought to this page because the emulator told you
-            that a new version is available, download the new version and
-            extract it over the existing folder. This way you won't lose
-            any save files that you may have.
-        </p>
+        <h2>Emulators supporting RetroAchievements</h2>
 
         <?php foreach ($emulators as $emulator): ?>
             <h2 class="longheader" id="<?= mb_strtolower($emulator['handle'] ?? null) ?>">
-                <a href="#<?= mb_strtolower($emulator['handle'] ?? null) ?>"><?= $emulator['handle'] ?? null ?></a> <small>(<?= $emulator['name'] ?? null ?>)</small>
-            </h2>
-            <div class="mb-1">
-                <?php if ($emulator['description'] ?? false): ?>
-                    <?= nl2br($emulator['description']) ?><br>
+                <a href="#<?= mb_strtolower($emulator['handle'] ?? null) ?>"><?= $emulator['handle'] ?? null ?></a>
+                <?php if ($emulator['handle'] != $emulator['name']): ?>
+                    <small>(<?= $emulator['name'] ?? null ?>)</small>
                 <?php endif ?>
-                <?php if ($emulator['link'] ?? false): ?>
-                    <a class="" href="<?= $emulator['link'] ?>">Documentation</a>
-                <?php endif ?>
-            </div>
-            <div class="mb-3 flex flex-col lg:flex-row justify-between items-start">
-                <div class="flex-1 mb-3">
-                    <?php if (!empty($emulator['systems'])): ?>
-                        <?php sort($emulator['systems']) ?>
-                        <b>Supported Systems:</b><br>
-                        <ul style="column-count: 3">
-                        <?php foreach ($emulator['systems'] as $system): ?>
-                            <?php
-                            sanitize_outputs($system);
-                            ?>
-                            <li>- <?= $system ?></li>
-                        <?php endforeach ?>
-                        </ul>
+            </h2>
+            <div class="flex flex-col lg:flex-row justify-between items-start">
+                <div class="mb-3">
+                    <?php if ($emulator['description'] ?? false): ?>
+                        <div class="mb-2"><?= nl2br($emulator['description']) ?></div>
```

Comment:
For some reason, moving the description inside the left column (instead of across the top of both column) breaks the horizontal fill on the bullet list. Any help to restore that would be appreciated.

Before:
![image](https://user-images.githubusercontent.com/32680403/232254057-62bf69f4-ffee-49fb-b8a2-696f663d2dce.png)

After:
![image](https://user-images.githubusercontent.com/32680403/232254077-3323a007-3841-4386-af04-9120a436d45c.png)